wgpu-core/src/device/resource.rs

```diff
@@ -1760,6 +1760,17 @@ impl Device {
                         _ => (),
                     }
                     match access {
+                        wgt::StorageTextureAccess::Atomic
+                            if !self.features.contains(
+                                wgt::Features::TEXTURE_ADAPTER_SPECIFIC_FORMAT_FEATURES
+                                    | wgt::Features::TEXTURE_ATOMIC,
```

Comment:
Do we really need both? Generally the features we have for textures don't require TASFF

Reply:
nah we dont, but we'll need a check in guaranteed format features and a couple tweaks